### PR TITLE
Remove CSRF implementation from Grape

### LINKED ISSUE #18: Fix the CSRF token usage in React powered forms
**What:** Fix the CSRF token usage in React powered forms

**Why:** We are having problems with CSRF tokens not working in React based forms. Right now there's code that implements how React is getting this token, but it is also generating the CSRF token manually, something that should be handled by Rails

**How:**

- Remove code to generate and inject CSRF and use Rails implementation instead.
- Make React powered forms work correctly.

### PR DESCRIPTION
**Pull request by orlando**
**What:** Remove CSRF implementation from Grape
 
**Why:** Close #18

**How:**

- Add standard gem for formatting
- Add instructions on how to configure formatting in VS Code
- Remove CSRF implementation in Grape.
- Add fonts to assets pipeline and fix font-face references in SCSS files

## Notes

Grape endpoint just have 2 endpoints to create and read non-profits

```ruby
GET  |  /v1/nonprofit/:id(.:format)      | Return a nonprofit.
POST |  /v1/nonprofit(.:format)          | Register a nonprofit
GET  |  /v1/swagger_doc(.:format)        | Swagger compatible API description
GET  |  /v1/swagger_doc/:name(.:format)  | Swagger compatible API description for specific API
```

I think this work was just starting and is not the direction we are moving now given we are implementing other apps with GraphQL APIs. We need to create follow up tickets for:

- Remove the Grape API and merge these actions (create and fetch a non-profit) to Rails controllers.
- Change React references to point to Rails controllers instead of Grape controllers.
- Remove Swagger, won't be needed since we won't use REST.

### CODE REVIEW

**Comment by orlando** [comment]
This is the follow up ticket to remove Grape https://github.com/debtcollective/fundraising/issues/27

File: Gemfile [hunk 1]
```diff
@@ -87,8 +94,10 @@ group :development, :ci, :test do
   gem 'dotenv-rails', '~> 2.7', '>= 2.7.5'
   gem 'mail_view', '~> 2.0'
   gem 'pry', '~> 0.12.2'
+  gem 'pry-byebug', '~> 3.7.0'
   gem 'ruby-prof', '0.15.9'
   gem 'solargraph', '~> 0.35.1'
+  gem 'standard', '~> 0.1.2'
```
Comment (castrolem) [inline Gemfile]: Totally needed.

File: Gemfile [hunk 2]
```diff
@@ -75,6 +74,14 @@ gem 'grape-swagger-entity', '~> 0.3.3'
 gem 'grape-swagger', '~> 0.33.0'
 gem 'grape', '~> 1.2', '>= 1.2.4'
 
+group :production do
+  gem 'rack-timeout', '~> 0.5.1'
```
Comment (castrolem) [inline Gemfile]: Good catch.
There's a `group :production do` at the end of the `Gemfile`, let's add this there so we group things in a more cohesive way than before.

File: docs/GETTING_STARTED.MD [hunk 3]
```diff
@@ -6,9 +6,9 @@
 
 You'll need to have in your Mac the following dependencies installed, if you don't want to use the provided Docker containers.
 
-* Ruby `2.5.1`
-* Rails `5.0.7.1`
-* Node `11.12.0`
+- Ruby `2.5.1`
```
Comment (castrolem) [inline docs/GETTING_STARTED.MD]: Nitpicky!

**Comment by castrolem** [comment]
@orlando I ran the specs and there's one (1) failing.

Let's add also a task to have a CI to run them and verify this on a PR level

**Comment by castrolem** [comment]
I fixed the spec, so I will merge this branch.

**Pull request merged**